external/easylogging++/easylogging++.h

```diff
@@ -2022,10 +2022,11 @@ class TypedConfigurations : public base::threading::ThreadSafe {
     if (it == confMap->end()) {
       try {
         return confMap->at(Level::Global);
-      } catch (...) {
+      } catch (std::exception & ex){
```

Comment:
Revert `catch (...)` and do a `throw;` instead, which rethrows the current exception.

Reply:
Done.